src/main/java/com/github/maxopoly/BottledExperience/PlayerListener.java

```diff
@@ -1,5 +1,9 @@
 package com.github.maxopoly.BottledExperience;
 
+import java.util.HashMap;
+
+import net.md_5.bungee.api.ChatColor;
```

Comment:
Oh crap, forgot to fix this before merging. Anyway, this uses the Bungee ChatColor; you should be using the Bukkit ChatColor.